common/lib/common-utils/src/trace.ts

```diff
@@ -3,15 +3,15 @@
  * Licensed under the MIT License.
  */
 
-import { performanceNow } from "./performanceNowNode";
+import { performance } from "./indexNode";
```

Comment:
Switched to `indexNode` so it will be correctly replaced to indexBrowser during webpack steps for isomorphism.